internal/sidekick/internal/rust/annotate.go

```diff
@@ -1100,6 +1098,46 @@ func (c *codec) annotateEnum(e *api.Enum, model *api.API, full bool) {
 		}
 	}
 
+	// We try to pick some good enum values to show in examples.
+	// - We pick from the already computed unique values, even though that applies to BigQuery only.
+	// - We pick values that are not deprecated.
+	// - We don't pick the default value.
+	var goodValues []*api.EnumValue
+	for _, ev := range unique {
```

Comment:
@coryan I didn't find slices.Filter or slices.Transform which would have been useful as well. It seems that's only available on experimental go packages? Or am I missing something.

I did rewrote this code some.

Reply:
Yep, done, using those is better, I hadn't found them.